photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/tuner/AtlasTuner.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.hyperparameter.tuner
+
+import breeze.linalg.DenseVector
+
+import com.linkedin.photon.ml.HyperparameterTuningMode
+import com.linkedin.photon.ml.HyperparameterTuningMode.HyperparameterTuningMode
+import com.linkedin.photon.ml.evaluation.Evaluator
+import com.linkedin.photon.ml.hyperparameter.EvaluationFunction
+import com.linkedin.photon.ml.hyperparameter.search.{GaussianProcessSearch, RandomSearch}
+
```

Comment:
Just to confirm, the lines 19-22 will stay in photon-ml as traits and 23 will move to atlas-ml. Correct?

Reply:
Yes. And the evaluator will be removed later.